frontend/src/utils/config.ts

```diff
@@ -1,7 +1,7 @@
 export const API_HOST = process.env.NEXT_PUBLIC_API_HOST;
 export const todoAPIEndpoints = {
-  list: () => `${API_HOST}/api/todo`,
-  create: () => `${API_HOST}/api/todo`,
-  update: (id: number) => `${API_HOST}/api/todo/${id}/`,
-  delete: (id: number) => `${API_HOST}/api/todo/${id}/`,
+  list: () => `${API_HOST}/v1.0/api/todo`,
+  create: () => `${API_HOST}/v1.0/api/todo`,
+  update: (id: number) => `${API_HOST}/v1.0/api/todo/${id}/`,
+  delete: (id: number) => `${API_HOST}/v1.0/api/todo/${id}/`,
```

Comment:
Frontend should be API independant so any backend implementation shouldn't require changes in the frontend

Reply:
Ive just saw this, srry jeje, THAT is a nice point of view and im in favour, but I also think that API independence is API gateway's responsibility (can be this an idea for another project? advanced api versioning 😄), I understand that it is not in the scope for this project so I can rollback it, thanks ✌🏾